tests/one_drive/fake_sheet.py

```diff
@@ -0,0 +1,140 @@
+FAKE_SHEET = {
```

Comment:
Can you call this file fixtures instead of fake_sheet ?
And it's where you're going to put all necessary fixtures for the test !